src/test/java/uk/gov/companieshouse/extensions/api/attachments/file/FileTransferApiClientUnitTest.java

```diff
@@ -176,10 +170,9 @@ public void testDownload_GenericException() {
         when(restTemplate.execute(eq(DOWNLOAD_URI), eq(HttpMethod.GET), any(RequestCallback.class), ArgumentMatchers.<ResponseExtractor<ClientHttpResponse>>any(), any(FileTransferApiClientResponse.class)))
             .thenThrow(exception);
 
-        expectedException.expect(RestClientException.class);
-        expectedException.expectMessage(exception.getMessage());
-
-        fileTransferApiClient.download(FILE_ID, servletResponse);
+        RestClientException thrown = assertThrows(RestClientException.class, () ->
+            fileTransferApiClient.download(FILE_ID, servletResponse));
+        assertEquals(thrown.getMessage(), thrown.getMessage());
```

Comment:
same as above:
assertEquals(thrown.getMessage(), thrown.getMessage()); will never be false, it's the same as saying does a == a
we need to check that the message in the thrown exception is what is expected
assertEquals(EXCEPTION_MESSAGE, thrown.getMessage());